### DISCUSSION TITLE
Imagick not working with GAE runtime php82 or php83

### DISCUSSION THREAD

**Discussion by mrceolla** [ImageMagick/ImageMagick#7624] [Help]
### ImageMagick version

7.?

### Operating system, version and so on

Google App Engine PHP 8.2 & 8.3

### Description

We have been using Imagick with Google App Engine with runtime php81 for some time. I changed the runtime in app.yaml php83 and received the following error when testing Imagick:

Uncaught ImagickException: no decode delegate for this image format `SVG' @ error/blob.c/BlobToImage/363 in /workspace/...

I tried runtime php82 and received the same error. Why is this?

I understand there may not yet be an official Imagick release for PHP 8.3, but there is for 8.2. Yet php82 doesn't work either. Did I do something wrong?

I am unsure of the versions used in Google App Engine.  imagick.so is an extension available in that environment by default so I did not need to install anything.

Image Magick sort of works. I can read PNG height and width with it when using runtime php82, but it cannot ingest SVG tags as it used to like when it was using runtime php81.

Thanks!

**Comment by dlemstra** [comment]
We don't maintain IMagick. You will need to create an issue or ask your question here instead: https://github.com/Imagick/imagick

**Reply by mrceolla** [reply]
My apologies.  Thank you for the direction

**Reply by dlemstra** [reply]
No worries and feel free to come back here if it's an ImageMagick issue.
